### ISSUE TITLE
Wrong hexadecimal generation number for e162236f

### ISSUE THREAD

**Issue by sheeryjay** [CuriosAI/sai#70]
It appears that e162236f got mistakenly marked as a g08b network, even though 84910f22 was already g08b network. Note that said error only affects hexadecimal generation number, not the decimal one.
This also probably caused the erroneous reference match 

> 019-12-04 17:2258.1 minutes ago | 84910f22 VS e162236f | 26 : 3 : 15 (62.50%) | 44 / 40 | reference

which should have been b12a3055 against e162236f. Curiously it appears to have somewhat weird result as well given that e162 should have won and not 8491.

**Comment by Vandertic** [comment]
Thank you! I noticed the match (and corrected it), but failed to notice what was the original problem!
Edit: Now everything should be ok. Thank you again for pointing this out.

**Comment by l1t1** [comment]
2019-12-04 13:37	b20af52a VS  84910f22
34 : 4 : 33 (50.70%)	71 / 50	promotion
2019-12-04 13:25	e162236f VS  84910f22
51 : 10 : 52 (49.56%)	113 / 50	promotion

these matches have too many games

**Comment by l1t1** [comment]
and The vs link of 113 games only shows 69 games.

**Comment by Vandertic** [comment]
You are right. All consequences of the original problem. I will see if these other errors have any consequence.
